Emby.Server.Implementations/News/NewsEntryPoint.cs

```diff
@@ -82,7 +82,7 @@ private async Task DownloadNews(string path)
 
             var requestOptions = new HttpRequestOptions
             {
-                Url = "https://emby.media/community/index.php?/blog/rss/1-media-browser-developers-blog",
+                Url = "https://github.com/jellyfin/jellyfin",
```

Comment:
I think this place warrants a `//TODO: add proper rss feed` comment or something similar